srcalc/src/main/java/gov/va/med/srcalc/domain/Patient.java

```diff
@@ -160,6 +164,19 @@ public void setLabs(final Map<String, RetrievedValue> labs)
         fLabs = labs;
     }
     
+    /**
+     * Returns the patient's health factors in String form.
```

Comment:
What does "String form" mean? Is this just the health factor name? Does it include the date?